esp/services/esdl_svc_engine/esdl_binding.cpp

```diff
@@ -65,29 +65,28 @@ bool trimXPathToParentSDSElement(const char *element, const char * xpath, String
     return false;
 }
 
-IPropertyTree * fetchESDLDefinitionFromDaliById(const char *id)
+void fetchESDLDefinitionFromDaliById(const char *id, StringBuffer & exsdl)
 {
     if (!id || !*id)
         throw MakeStringException(-1, "Unable to fetch ESDL Service definition information, service id is not available");
 
-    DBGLOG("ESDL Binding: Fetching ESDL Definition from Dali: %s ", id);
+    //There shouldn't be multiple entries here, but if so, we'll use the first one
+    VStringBuffer xpath("%s[@id='%s'][1]", ESDL_DEF_ENTRY, id);
+
+    ESPLOG(LogNormal, "ESDL Binding: Fetching ESDL Definition from Dali: %s ", id);
```

Comment:
Good point Jake, I actually went back and forth on this one, but ultimately chose to log it because it's confirmation of an important event, and it shouldn't happen often (once per service at load time, and once per update to the service interface at run-time.